### DISCUSSION TITLE
如何修改日志格式，使其宽度相等

### DISCUSSION THREAD

**Discussion by chrisyy2003** [nonebot/discussions#269] [问答] [answered]
如何修改日志format，从而使得各种日志等级为相同的宽度
![image](https://user-images.githubusercontent.com/63803385/207207106-4d9492e2-2f52-4e93-b777-5e69afc25747.png)

![image](https://user-images.githubusercontent.com/63803385/207206976-ecfb380e-55a5-4d97-9f42-5347a1650958.png)

效果像这样
![image](https://user-images.githubusercontent.com/63803385/207207290-4eefb3bd-ccdc-4721-9936-c5a96ba06d8c.png)

**Comment by yanyongyu** [comment, marked as answer]
参考https://v2.nonebot.dev/docs/tutorial/custom-logger ，自行修改logger的format

```python
logger.remove()
logger.add(xxxx)
```

**Reply by chrisyy2003** [reply]
已解决，感谢
![image](https://user-images.githubusercontent.com/63803385/207214897-77c91321-0d34-4524-8cab-d2be585bd6b7.png)
